src/content/TokenSigning/injectPageScript.ts

```diff
@@ -40,7 +40,13 @@ export default function injectPageScript(): void {
     s.type = "text/javascript";
     s.dataset.name = "TokenSigning";
     s.dataset.by = "Web-eID extension";
-    s.innerHTML = "(" + pageScript + ")();";
+
+    if (chrome.runtime.getManifest()["manifest_version"] === 3) {
```

Comment:
```suggestion
    if (browser.runtime.getManifest()["manifest_version"] === 3) {
```

Also, consider reversing the `if` condition so that in case of manifest version 2 we use `innerHTML`, else `src` assignment.

Reply:
On a second thought, maybe reversing the manifest version check is not a good idea after all as in case of quirks `innterHTML` is a safer bet.